src/Resources/ResourceManager/Implementation/CmdletBase/DeploymentCmdletBase.cs

```diff
@@ -84,13 +90,15 @@ protected ResourceWithParameterCmdletBase()
         public Hashtable TemplateParameterObject { get; set; }
 
         [Parameter(ParameterSetName = TemplateObjectParameterFileParameterSetName,
-            Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "A file that has the template parameters.")]
+            Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Parameter file to use for the template.")]
         [Parameter(ParameterSetName = TemplateFileParameterFileParameterSetName,
-            Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "A file that has the template parameters.")]
+            Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Parameter file to use for the template.")]
```

Comment:
Please update parameter description in MDs as well

Reply:
👍